ui/apps/platform/src/Containers/AccessControl/PermissionSets/ResourceDescription.tsx

```diff
@@ -26,13 +30,17 @@ const resourceDescriptions: Record<ResourceName, string> = {
     DebugLogs:
         "Read: View the current logging verbosity level of all components, including Central, Scanner, Sensor, Collector, and Admission controller. Download diagnostic bundle. Note: The diagnostic bundle contains information about all clusters and namespaces regardless of the user's role and access scope. Don't give this permission to users with limited access scope. Write: Modify the logging verbosity level.",
     Deployment: 'Read: View deployments (workloads) in secured clusters. Write: N/A',
+    DeploymentExtensions:
+        'Read: View risk score and baseline extensions of deployments. Write: Modify the baseline extensions of deployments.',
```

Comment:
also process activity? I'd also spell out which baselines are meant: `network and process baselines`.

Reply:
Addressed in follow-up PR.